dojo/unittests/tools/test_pmd_parser.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+from dojo.tools.pmd.parser import PmdParser
+from dojo.models import Test
+
+
+class TestPMDParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_no_vuln.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
```

Comment:
Add more test on attributes like:
```suggestion
        self.assertEqual("Medium", findings[0].severity)

```